frontend/src/component/demo/Demo.tsx

```diff
@@ -1,52 +1,57 @@
-import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
 import { useEffect, useState } from 'react';
 import { DemoTopics } from './DemoTopics/DemoTopics';
 import { DemoSteps } from './DemoSteps/DemoSteps';
-import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
 import { createLocalStorage } from 'utils/createLocalStorage';
 import { TOPICS } from './demo-topics';
 import { DemoDialogWelcome } from './DemoDialog/DemoDialogWelcome/DemoDialogWelcome';
+import { DemoDialogFinish } from './DemoDialog/DemoDialogFinish/DemoDialogFinish';
 
 const defaultProgress = {
     welcomeOpen: true,
     expanded: true,
-    active: false,
-    topic: 0,
+    topic: -1,
     steps: [0],
 };
 
 const { value: storedProgress, setValue: setStoredProgress } =
     createLocalStorage('Tutorial:v1', defaultProgress);
 
 export const Demo = () => {
-    const { uiConfig } = useUiConfig();
     const [welcomeOpen, setWelcomeOpen] = useState(
         storedProgress.welcomeOpen ?? true
     );
-    const [active, setActive] = useState(false);
+    const [finishOpen, setFinishOpen] = useState(false);
+
     const [expanded, setExpanded] = useState(storedProgress.expanded ?? true);
-    const [topic, setTopic] = useState(storedProgress.topic ?? 0);
+    const [topic, setTopic] = useState(storedProgress.topic ?? -1);
     const [steps, setSteps] = useState(storedProgress.steps ?? [0]);
 
-    useEffect(() => {
-        if (storedProgress.active) {
-            setTimeout(() => {
-                setActive(true);
-            }, 1000);
-        }
-    }, []);
-
     useEffect(() => {
         setStoredProgress({
             welcomeOpen,
             expanded,
-            active,
             topic,
             steps,
         });
-    }, [welcomeOpen, expanded, active, topic, steps]);
+    }, [welcomeOpen, expanded, topic, steps]);
 
-    if (!uiConfig.flags.demo) return null;
+    const onStart = () => {
+        setTopic(0);
+        setSteps([0]);
```

Comment:
On the topic field, I'm curious why it's not `-1` as in the default. Maybe it's just that zero is the first topic? I'm tempted to suggest `setTopic(defaultProgress.topic + 1);` but that's an assumption that the default topic is set to -1 and feels like unnecessary coupling, so I don't like it, but: 
```suggestion
        setTopic(0); 
        setSteps(defaultProgress.steps);
```